File: Models/Player.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace sclask.Models
{
  public class Player
  {
    public int Id { get; set; }
    [StringLength(255)]
    [Required]
    public string FullName { get; set; }
    [EmailAddress]
    [Required]
    public string EmailAddress { get; set; }
    [Url]
    public string ProfilePhoto { get; set; }
  }
}

File: ClientApp/src/models/Player.js
export class Player {
  constructor ({
    id = undefined,
    fullName = '',
    emailAddress = '',
    profilePhoto = ''
  } = {}) {
    this.id = id;
    this.fullName = fullName;
    this.emailAddress = emailAddress;
    this.profilePhoto = profilePhoto;
  }

  toJSON = () => {
    return JSON.stringify({
      id: this.id,
      fullName: this.fullName,
      emailAddress: this.emailAddress,
      profilePhoto: this.profilePhoto
    })
  }
}